<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskTransformDialog</class>
 <widget class="QDialog" name="AskTransformDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenBRF</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Rotation</string>
   </property>
   <widget class="MyDoubleSpinBox" name="rotx">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>20</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="MyDoubleSpinBox" name="roty">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>40</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="MyDoubleSpinBox" name="rotz">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>60</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>21</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>41</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>61</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Translation</string>
   </property>
   <widget class="MyDoubleSpinBox" name="trax">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>20</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="MyDoubleSpinBox" name="tray">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>40</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="MyDoubleSpinBox" name="traz">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>60</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>21</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>41</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>61</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QPushButton" name="rightX">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>1</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/ico_right.png</normaloff>icons/ico_right.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="leftX">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/ico_left.png</normaloff>:/icons/ico_left.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="midX">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>1</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/ico_mid.png</normaloff>:/icons/ico_mid.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QPushButton" name="leftY">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/ico_left.png</normaloff>:/icons/ico_left.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="midY">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>1</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/ico_mid.png</normaloff>:/icons/ico_mid.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="rightY">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>1</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/ico_right.png</normaloff>icons/ico_right.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QPushButton" name="leftZ">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/ico_left.png</normaloff>:/icons/ico_left.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="midZ">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>1</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/ico_mid.png</normaloff>:/icons/ico_mid.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="rightZ">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>1</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/ico_right.png</normaloff>icons/ico_right.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Scale %</string>
   </property>
   <widget class="MyDoubleSpinBox" name="scx">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>20</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="MyDoubleSpinBox" name="scy">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>40</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="MyDoubleSpinBox" name="scz">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>60</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>21</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>41</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>61</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>52</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Uniform</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="applyToLastSel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply to last selected
object only</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../tablemodel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>trax</tabstop>
  <tabstop>leftX</tabstop>
  <tabstop>midX</tabstop>
  <tabstop>rightX</tabstop>
  <tabstop>tray</tabstop>
  <tabstop>leftY</tabstop>
  <tabstop>midY</tabstop>
  <tabstop>rightY</tabstop>
  <tabstop>traz</tabstop>
  <tabstop>leftZ</tabstop>
  <tabstop>midZ</tabstop>
  <tabstop>rightZ</tabstop>
  <tabstop>rotx</tabstop>
  <tabstop>roty</tabstop>
  <tabstop>rotz</tabstop>
  <tabstop>scx</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>scy</tabstop>
  <tabstop>scz</tabstop>
  <tabstop>applyToLastSel</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AskTransformDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AskTransformDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
